lib/chef/mixin/user_context.rb

```diff
@@ -22,7 +22,8 @@ class Chef
   module Mixin
     module UserContext
 
-      def with_user_context(user, password, domain = nil, &block)
+      # valid logon_type values => :remote, :local
```

Comment:
Please add a comment like, "logon_type specifies if we use the credentials to create a logon session against the local system, and then try to access the files in the case of :local, or if we continue with the current user but pass the provided credentials to the remote system with :remote"